tests/test_static_check.py

```diff
@@ -22,4 +22,4 @@ def test_black() -> None:
 
 def test_pylint() -> None:
     """Lint check"""
-    run(["pylint"] + PKGS, check=True)
+    run(["pylint"] + ["--load-plugins", "pylint_pytest"] + PKGS, check=True)
```

Comment:
I think a better strategy for this would be to have a pyling configuration file at the root of our repo. That way it will work even when running pylint manually or via an IDE.

Reply:
Yes, It's an option. can do

Reply:
Currently it is not working as addressed in documentation 
`pylint-pytest` suggest this:
```
[MASTER]
load-plugins=pylint_pytest
```
but it doesn't work (used an example from their docs)

pylint documentation suggest using `[MAIN]`, but that doesn't work as well.

Reply:
I seem to remember we used to have a Pylint configuration file somewhere, perhaps in the PCI repo, check it out

Reply:
We did not use pylint configuration file, but we did apply flags on runtime with `test_qa.py` (on vanilla pylint, no plugin).

Reply:
Check [here](https://pylint.pycqa.org/en/latest/user_guide/usage/run.html) for how to include PyLint configuration in `pyproject.toml`, which is what you probably should do since this project has that file.

It seems that the section name aught to be `MAIN` or `tool.pylint.'MAIN'` in `pyproject.toml`

Reply:
Updated, working now. 
It should be noted that this plugin helps with very specific errors; we will still need to have docstrings for each test function and module. 
ways to solve it are either ignoring those errors for the whole tests directory (cons: there might be multiple tests at multiple directories later on)
or adding ignores at the correct location/top of each test file (con: ugly).